src/zappy_server_src/core/commands/join_command.c

```diff
@@ -71,7 +72,7 @@ bool command_join(char *arg, player_list_t *plr, server_data_t *serv)
     plr->player_data = player_entity;
     entity_wrapper_add_player(serv->entities, player_entity);
     add_user_to_team(tmp, ((player_t *)player_entity->data)->uuid);
-    send_entities_list_info(serv);
+    entity_diff_add_entity(serv->modified_entities, player_entity);
```

Comment:
Useless, the entity is already add to the list in command_join_create_player_data function.